test/pangolin/pangolin.test.js

```diff
@@ -0,0 +1,204 @@
+const { expect } = require("chai");
+const hre = require("hardhat");
+
+const { web3, deployments, waffle, ethers } = hre;
+const { provider, deployContract } = waffle
+
+const abis = require("../../scripts/constant/abis");
+const buildDSAv2 = require("../../scripts/buildDSAv2_avalanche")
+const deployAndEnableConnector = require("../../scripts/deployAndEnableConnector.js")
+const encodeSpells = require("../../scripts/encodeSpells.js")
+const getMasterSigner = require("../../scripts/getMasterSigner_avalanche")
+
+const connectV2PangolinArtifacts = require("../../artifacts/contracts/avalanche/connectors/pangolin/main.sol/ConnectV2PngAvalanche.json");
```

Comment:
```suggestion
import { ConnectV2PngAvalanche__factory } from "../../../typechain";
```

Reply:
This file was moved to pangolin_exchange.test.ts, please ignore this file, It was a mistake of mine not erased it